### PR TITLE
Vishnupsingh523 patch 1

### PR DESCRIPTION
**Pull request by vishnupsingh523**
please add me

### CODE REVIEW

**Comment by SparshPrajapati** [comment]
its nice that you are learnin somethng new
 👍 💯

File: action-a/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,5 @@
+FROM debian:9.5-slim
```
Comment (adityasinghpratham) [inline action-a/Dockerfile:1]: its correct

Comment (vishnupsingh523) [reply action-a/Dockerfile:1]: good

**Comment by vishnupsingh523** [comment]
thanks

**Pull request closed without merging**